<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation Run Time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="runTimeLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="landTimeLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Planes Landing Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgLbl">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CrowdedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>1201</width>
      <height>571</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1199</width>
       <height>569</height>
      </rect>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1181</width>
        <height>541</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1181</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="CurrentTimeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Time:: 00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="Event">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>scroll</zorder>
   <zorder>label_2</zorder>
   <zorder>runTimeLabel</zorder>
   <zorder>landTimeLabel</zorder>
   <zorder>label_3</zorder>
   <zorder>startButton</zorder>
   <zorder>avgLbl</zorder>
   <zorder>CrowdedLabel</zorder>
   <zorder>CurrentTimeLabel</zorder>
   <zorder>Event</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
